app/src/main/java/com/github/polypoly/app/ui/map/MapViewModel.kt

```diff
@@ -2,14 +2,15 @@ package com.github.polypoly.app.ui.map
 
```

Comment:
since this file no longer depends on UI stuff, it could be moved to the package where it should be instead of `ui`?